prdoc/pr_5430.prdoc

```diff
@@ -0,0 +1,13 @@
+# Schema: Polkadot SDK PRDoc Schema (prdoc) v1.0.0
+# See doc at https://raw.githubusercontent.com/paritytech/polkadot-sdk/master/prdoc/schema_user.json
+
+title: "pallet-collator-selection: use correctly the `WeightInfo` call for `new_session`"
+
+doc:
+  - audience: Runtime Dev
+    description: |
+      - Fixes an incorrect usage of the `WeightInfo` trait for `new_session`.
+
+crates:
+  - name: pallet-collator-selection
+    bump: minor
```

Comment:
This is a patch, not a minor change. The API is the same in all regards, this is a transparent change for users.

Reply:
Thanks for the tip!